features.html

```diff
@@ -1,215 +1,80 @@
-          </div>
-          <div>
-            <a href="#" language="ca">Català</a>
-          </div>
-          <div>
-            <a href="#" language="es">Español</a>
-          </div>
-        </div>
+<div class="goteo">
+  <div class="row image-block">
+    <div class="col-lg-6 widget text-center">
+      <iframe frameborder="0" height="492px" src="https://www.goteo.org/widget/project/katuma" width="300px" scrolling="no"></iframe>
+    </div>
+    <aside class="col-lg-5">
+      <h2 data-i18n="features.goteo.title">Ens ajudes a desenvolupar noves funcionalitats?</h2>
+      <p data-i18n="features.goteo.body">Amb aquesta campanya a Goteo seguirem desenvolupant Katuma, acompanyarem a grups i productores en la seva implantació i millorarem la nostra comunicació perquè Katuma arribi a tothom.</p>
+      <div class="button-group">
+        <a onClick="trackOutboundLink('boton', 'click', 'Matchfunding2018-widget','https://www.goteo.org/project/katuma'); return false;" class="button negative-button" data-i18n="goteo.button" href="https://www.goteo.org/project/katuma">Fes una donació</a>
+        <a class="button secondary-button login" data-i18n="goteo.secondary-button" href="https://alpha.katuma.org/">Entra</a>
       </div>
-    </div><!-- /container -->
-    <p>
-    </p>
-  </div><!--footer-->
-
-  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
-  <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.10.1.min.js"><\/script>')</script>
-
-  <script src="js/vendor/i18next-1.7.2.min.js"></script>
-
-  <script src="js/main.js"></script>
-
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-    ga('create', 'UA-48794646-1', 'auto');
-    ga('send', 'pageview');
-  </script>
-  <script>
-    /**
-    * Función de Analytics que realiza un seguimiento de un clic en un enlace externo.
-     * Esta función toma una cadena de URL válida como argumento y la utiliza
-     * como la etiqueta del evento. Configurar el método de transporte como "beacon" permite que el hit se envíe
-     * con "navigator.sendBeacon" en el navegador que lo admita.
-     */
-    var trackOutboundLink = function(event_category, event_type, event_label, url) {
-      ga('send', 'event', event_category, event_type, event_label+'-'+url, {
-        'transport': 'beacon',
-        'hitCallback': function(){document.location = url;}
-      });
-    }
-
-  </script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
-  </body>
-</html>
+    </aside>
+  </div>
+</div>
```

Comment:
OMG you level up 💪 

But Jekyll is not cool anymore. You could use  https://www.gatsbyjs.org/ it use React and GraphQL 💯

Reply:
*ayparfavar* @andresgutgon :trollface: